src/helpers/manage-merge-queue.ts

```diff
@@ -83,13 +83,26 @@ export const removePrFromQueue = async (pullRequest: PullRequest) => {
 };
 
 const addPrToQueue = async (pullRequest: PullRequest, queuePosition: number) =>
-  octokit.issues.addLabels({
-    labels: [`${QUEUED_FOR_MERGE_PREFIX} #${queuePosition}`],
-    issue_number: context.issue.number,
-    ...context.repo
-  });
+  Promise.all([
+    octokit.issues.addLabels({
+      labels: [`${QUEUED_FOR_MERGE_PREFIX} #${queuePosition}`],
+      issue_number: context.issue.number,
+      ...context.repo
+    }),
+    enableAutoMerge(pullRequest.id)
+  ]);
 
 const getQueuedPullRequests = async (): Promise<PullRequestList> => {
   const openPullRequests = await paginateAllOpenPullRequests();
   return openPullRequests.filter(pr => pr.labels.some(label => label.name === READY_FOR_MERGE_PR_LABEL));
 };
+
+export const enableAutoMerge = async (pullRequestId: number, mergeMethod = 'SQUASH') => {
+  return octokitGraphql(`
+    mutation {
+      enablePullRequestAutoMerge(input: { pullRequestId: ${pullRequestId}, mergeMethod: "${mergeMethod}" }) {
```

Comment:
i'm pretty sure the graphql mutations use some internal `id` separate from the pull request number, so you may have to do something like this first:
```gql
query {
   repository(name: "${context.repo.name}", owner: "${context.repo.owner}") {
      pullRequest(number: ${pullRequestId}) { id }
   }
}
```
and then use that `id` in the mutation

Reply:
looked into it and you can actually use `pullRequest.node_id`